CheckValues/Results.cs

```diff
@@ -15,14 +15,27 @@ namespace CheckValues
 {
     public class Results
     {
-
         public MonitorsController getController()
         {
+            string connection = "";
+            var _config = new ConfigurationBuilder()
+            .AddJsonFile("appsettings.json")
+            .Build();
             var builder = new ConfigurationBuilder();
             builder.AddUserSecrets<Startup>();
             var config = builder.Build();
+
+            if (config.GetSection("LocalLiveDBConnectionString").Exists())
+            {
+                connection = config.GetSection("LocalLiveDBConnectionString").Value;
+            }
+            else
+            {
+                connection = _config.GetSection("LocalLiveDBConnectionString").Value;
+            }
+
```

Comment:
Not sure what the use is of the if statement here. It checks if the LiveDB string is populated and then uses it in both the true and false blocks

Reply:
This basically checks if a appsettings.json "LiveLocalDBConnectionString" string exists to connect to otherwise it uses the UserSecrets.

Reply:
Maybe rename it to not differentiate between the two just with an underscore. (did not see that line)

Reply:
I have done so.